src/credentials/tests/CHIPCert_test_vectors.cpp

```diff
@@ -169,6 +169,43 @@ CHIP_ERROR GetTestCertPubkey(uint8_t certType, const uint8_t ** certPubkey, uint
     return err;
 }
 
+CHIP_ERROR GetTestCertAKID(uint8_t certType, ByteSpan & akid)
```

Comment:
This really seems like it would be much more scalable to have a utility to obtain the AKID/SKID. Not for this PR, but to be considered...